### ISSUE TITLE
Webpages caching

### ISSUE THREAD

**Issue by lordratner** [nitnelave/lldap#392]
Hi, 

I'm not 100% sure on the cause of this issue, but I *think* it's from the pages being cached in the browser.

There have been several instances where I've had to refresh, or in some cases force-refresh (ctrl-F5) go get the webui to work. SOme examples:

The login page was blank after switching from stable --> latest. Refreshing a hundred times did nothing. I completely destroyed the container and rebuilt. No luck. Only when I hit ctrl-f5 did it work.

Multiple times the user dropdown in the  upper right will do nothing until I refresh the page. Then it works. I can recreate this one every time. Logout, log back in, and boom, the menu won't work until I refresh. Doesn't matter how many other pages I go to. Chrome browser

Request a password reset. When it says a reset token has been sent, hit the "Back" button to go back to the login page. Try to log in. The error "Could not log in" will show no matter how many times you hit the button, or retype the password. Refresh the page and it works.

**Comment by nitnelave** [comment]
Yeah, caching can be an issue when changing versions of the frontend. To be honest, I'm not sure how that's normally handled.

However, I'm surprised about the dropdown, I'll have to investigate.

**Comment by lordratner** [comment]
Another issue with the dropdown. If I go a not-me user's details. Use the upperright menu for my user and click View Details. The URL in the address bar will change, but the actual page will not. Still on the not-me user's details. Still able to change details for that user. Hit refresh, and boom, I'm in my details.

**Comment by nitnelave** [comment]
I just tried debugging the dropdown issue, and it's very weird, because the component asks to be redrawn, but it's not. I don't want to report that as a bug to the framework, because I'm using a quite old version.

I guess the way forward is to upgrade through the breaking changes, but it's a non-trivial endeavor.

**Comment by Cajuteq** [comment]
Just my 2 cent, but I can confirm the dropdown bug also happens on Firefox and Edge
